Cubical/Categories/Adjoint.agda

```diff
@@ -0,0 +1,296 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+module Cubical.Categories.Adjoint where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Data.Sigma
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor
+open import Cubical.Categories.NaturalTransformation
+open import Cubical.Foundations.Isomorphism
+
+open Functor
+
+open Iso
+open Precategory
+
+{-
+==============================================
+                  Overview
+==============================================
+
+This module contains two definitions for adjoint
+functors, and functions witnessing their
+logical (and maybe eventually actual?)
+equivalence.
+-}
+
+private
+  variable
+    ℓC ℓC' ℓD ℓD' : Level
+
+-- I tried using this... but Agda can never infer the category,
+-- even when it's pretty clear from context.....
+-- _⋆_ : ∀ {ℓ ℓ'} {C : Precategory ℓ ℓ'} {x y z} (f : C [ x , y ]) (g : C [ y , z ]) → C [ x , z ]
+-- _⋆_ {C = C} = seq C
+
```

Comment:
Yeah, it's not really possible to infer the category from the other arguments because they only refer to the field for the type of morphisms, so the rest of the category is ambiguous.

Reply:
Yeah, you can remove this commented code now

Reply:
Ah got it, thanks. Removed the comment